src/Classes/Canvas.ts

```diff
@@ -3,55 +3,89 @@
 import {Constants} from "./Utils/Constants";
 import {Container} from "./Container";
 import {PolygonMaskType} from "../Enums/PolygonMaskType";
+import {IDimension} from "../Interfaces/IDimension";
 
+/**
+ * Canvas manager
+ */
 export class Canvas {
+    /**
+     * The particles canvas
+     */
     public element: HTMLCanvasElement;
+    /**
+     * The particles canvas context
+     */
     public context: CanvasRenderingContext2D | null;
-    public width: number;
-    public height: number;
+    /**
+     * The particles canvas dimension
+     */
+    public dimension: IDimension;
+    /**
+     * The particles canvas container element id
+     */
     public tagId: string;
-    public pxratio: number;
-
+    /**
+     * The ratio used by the particles canvas
+     */
+    public pxRatio: number;
+
+    /**
+     * The parent container
+     */
     private readonly container: Container;
 
+    /**
+     * Constructor of canvas manager
+     * @param container the parent container
+     * @param tagId the particles container element id
+     */
     constructor(container: Container, tagId: string) {
         const canvasEl = document.querySelector(`#${tagId} > .${Constants.canvasClass}`) as HTMLCanvasElement;
 
         this.container = container;
         this.element = canvasEl;
-        this.width = canvasEl.offsetWidth;
-        this.height = canvasEl.offsetHeight;
+        this.dimension = {
+            width: canvasEl.offsetWidth,
+            height: canvasEl.offsetHeight
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'height' is not sorted alphabetically](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5110920)